.travis.yml

```diff
@@ -2,7 +2,10 @@ language: go
 go:
 - 1.9
 - "1.10"
+- "1.11.x"
 - tip
 matrix:
+  # todo: remove this when Go1.12 is released
+  - GO111MODULE=on
```

Comment:
Wouldn't this be better accomplished with a version matrix?

```
matrix:
  include:
    - go: '1.11.x'
      env: GO111MODULE=on
```

Which would allow Go 1.11 to be tested consistently while it's supported.